### ISSUE TITLE
Thumbnail is wrong when image name has multiple dot in filename

### ISSUE THREAD

**Issue by elcolie** [haltakov/simple-photo-gallery#126]
**Background:**<br>
I am running grid search of my A.I. generated pictures. My pictures contains multiple dots in filename. For example
`character_2.0_0.2_1.4000000000000001_8.png`

**Problem:**<br>
`gallery-build` omit the dot therefore my thumbnails are wrong. My thumbnails are sharing the same `character_2.jpg`

**Expected Result:**<br>
`gallery-build` smart enough to understand gridsearch and not sharing the same thumbnail.

Root cause comes from here.
```python
def get_thumbnail_name(thumbnails_path, photo_name):
    """
    Generates the full path to a thumbnail file
    :param thumbnails_path: Path to the folders where the thumbnails will be stored
    :param photo_name: Name of the original photo
    :return: Full path to the thumbnail file
    """
    photo_name_without_extension = os.path.basename(photo_name).split(".")[0]
    return os.path.join(thumbnails_path, photo_name_without_extension + ".jpg")
```

https://github.com/haltakov/simple-photo-gallery/blob/master/simplegallery/logic/variants/files_gallery_logic.py#L26

**Comment by kroesche** [comment]
I reproduced this

**Comment by kroesche** [comment]
addressed in release 1.5.6
